pylint_extra_requirements.txt

```diff
@@ -2,3 +2,4 @@
 
 # Write extra requirements for running pylint here:
 pytest
+PyGObject
```

Comment:
I am not sure if this is the best place for this "dependency", but pylint is complaining about import error on the `import gi` line. This is dependency of blivet so it is available on systems with the role (and the test suite) running, but not for the pylint run.

Reply:
The network role also uses `import gi` and pylint tests pass there.

Reply:
Network role uses it in a try-except block so I guess pylint is OK with that. Anyway installing PyGObject from pypi doesn't work, it's a C extension and it fails to compile. I think skipping the warning is probably the best way to "fix" this?